contrib/kube-prometheus/hack/cluster-monitoring/deploy

```diff
@@ -4,7 +4,7 @@ manifest_prefix=${1-.}
 
```

Comment:
I know this is not part of this PR, but would you mind setting _command failure_, _pipefail_, ... at the top of this file?

e.g. see: https://github.com/coreos/prometheus-operator/blob/master/scripts/travis-e2e.sh#L2

Reply:
Thanks!